docs/changelog/0037-update-supervisor-confs.md

```diff
@@ -0,0 +1,28 @@
+<!--THIS FILE IS AUTOGENERATED: DO NOT EDIT-->
+<!--See https://github.com/dimagi/commcare-cloud/blob/master/changelog/README.md for instructions-->
+# 37. Run command to update Supervisor configurations
+
+**Date:** 2020-10-14
+
+**Optional per env:** _only required on some environments_
+
+
+## CommCare Version Dependency
+The following version of CommCare must be deployed before rolling out this change:
+[05ebc51b](https://github.com/dimagi/commcare-hq/commit/05ebc51ba4b16a420f0fd64c17ffad89501ea36d)
+
+
+## Change Context
+Run management command to remove unused errand-boy processes.
+
+## Details
+The `errand-boy` package is no longer required. It's corresponding process
+group should be removed to avoid deploy errors when restarting services
+when `errand-boy` is not installed.
+
+## Steps to update
+Run the following management command to update supervisor configs:
+
+```bash
+cchq <env> --control update-supervisor-confs
```

Comment:
I would drop the `--control` since we don't know where the user will run the command.

Reply:
Thanks. https://github.com/dimagi/commcare-cloud/pull/4268/commits/13190aca9f6874a0edd46b14952ca15db4c5af93